tigers-lottery/app/src/main/java/com/example/tigers_lottery/DatabaseHelper.java

```diff
@@ -783,4 +813,14 @@ public void entrantAcceptDeclineInvitation(int eventId, String action, StatusCal
             Log.e("Firestore", "Error retrieving document");
         });
     }
+
+
+    public boolean validatingUserProfileInput(String firstName, String lastName, String email, Date date) {
+        return (Objects.equals(firstName, "") || Objects.equals(lastName, "") || Objects.equals(email, "") || date == null);
+    }
+
+
+    public boolean validatingFacilityProfileInput(Context context, String name, String email, String location, String mobile) {
+        return (!Objects.equals(name, "") && !Objects.equals(email, "") && !Objects.equals(location, "") && mobile != null);
+    }
```

Comment:
I dont think this code should be here, move it to the file you were calling this